sickrage/providers/nzb/NZBProvider.py

```diff
@@ -39,23 +39,13 @@ def _get_result(self, episodes):
         return NZBSearchResult(episodes)
 
     def _get_size(self, item):
-        try:
-            size = item.get('links')[1].get('length', -1)
-        except (AttributeError, IndexError, TypeError):
-            size = -1
```

Comment:
Actually they can raise any of those there, I removed them just to test.  I was more concerned about the other try-excepts but figured I would try removing this one too.